ims/people/confidentiality_agreements.md

```diff
@@ -0,0 +1,17 @@
+# Confidentiality Agreements
+
+This outlines Made Tech's policy in regards to confidentiality agreements.
+
+## The Policy
+
+ - All employees sign Terms and Conditions of Employment which include clauses for 
+   - Confidentiality
+   - Intellectual property
+ - Company Non-Disclosure Agreements (NDA’s) are included in contracts raised with contractors 
+ - Confidentiality requirements are also written into supplier contracts and agreements
+ - Visitors are not allowed in areas where sensitive information is accessible unless they are working under a current NDA
```

Comment:
Is this currently being enforced?

Reply:
Yep - John from SW has an NDA. Colin our ISO consultant does as well. 
For other visitors (not regularly in office) the rules around physical security and visitors apply.